controllers/common.go

```diff
@@ -1475,15 +1486,25 @@ func (r *MultiClusterHubReconciler) removePluginFromConsole(multiClusterHub *ope
 		return ctrl.Result{}, nil
 	}
 
+	modifiedSpec := false
 	// Remove mce to the plugins list if it is not already there
 	if utils.Contains(console.Spec.Plugins, "acm") {
 		console.Spec.Plugins = utils.RemoveString(console.Spec.Plugins, "acm")
+		modifiedSpec = true
+	}
+	// Remove mce to the plugins list if it is not already there
+	if utils.Contains(console.Spec.Plugins, "mce") {
+		console.Spec.Plugins = utils.RemoveString(console.Spec.Plugins, "mce")
+		modifiedSpec = true
+	}
+
+	if modifiedSpec {
 		err = r.Client.Update(ctx, console)
 		if err != nil {
-			log.Info("Failed to remove acm consoleplugin to console")
+			log.Info("Failed to remove consoleplugin from console")
 			return ctrl.Result{Requeue: true}, err
 		} else {
-			log.Info("Removed acm consoleplugin to console")
+			log.Info("Removed consoleplugin from console")
```

Comment:
Do not indent normal code when handling errors:
https://gist.github.com/adamveld12/c0d9f0d5f0e1fba1e551#indent-error-flow